app/client/components/helpCentre/knownIssues.tsx

```diff
@@ -104,7 +103,9 @@ export const KnownIssues = () => {
   `;
 
   const aCss = css`
+    ${textSans.medium()};
     text-decoration: underline;
+    color: black;
```

Comment:
Is there a palette value for this?

Reply:
Ah yes, will update to the source palette